src/net/sourceforge/kolmafia/textui/parsetree/CompositeValue.java

```diff
@@ -63,34 +63,23 @@ public Value aref( final Value key )
 		return this.aref( key, null );
 	}
 
-	public Value aref( final Value key, final AshRuntime interpreter )
-	{
-		return null;
-	}
+	public abstract Value aref( final Value key, final AshRuntime interpreter );
 
 	public void aset( final Value key, final Value val )
 	{
 		this.aset( key, val, null );
 	}
 
-	public void aset( final Value key, final Value val, final AshRuntime interpreter )
-	{
-	}
+	public abstract void aset( final Value key, final Value val, final AshRuntime interpreter );
 
-	public Value remove( final Value key, final AshRuntime interpreter )
-	{
-		return null;
-	}
+	public abstract Value remove( final Value key, final AshRuntime interpreter );
 
 	@Override
 	public void clear()
```

Comment:
This feels like it should also be abstract, since it doesn't do anything useful or generic.